test/autests/gold_tests/field_verification/not_nocase.test.py

```diff
@@ -0,0 +1,187 @@
+'''
+Verify correct field verification behavior for not and nocase.
+'''
+# @file
+#
+# Copyright 2021, Verizon Media
+# SPDX-License-Identifier: Apache-2.0
+#
+
+
+Test.Summary = '''
+Verify correct field verification behavior for
+equals, absent, present, contains, prefix, and suffix
+with not, nocase, and both not and nocase modifiers
+'''
+
+#
+# Test 1: Verify field verification in a YAML replay file.
+# Each combinaton of test type, not/as, and case/nocase, and positive/negative result
+# are tested for client, and a mixture for server
+#
+r = Test.AddTestRun("Verify field verification works for a simple HTTP transaction")
```

Comment:
Also URL verification. And the emphasis is on no/prefix. Maybe the following?

"Verify 'not' and 'nocase' directives work for a single HTTP transaction."